packages/client-python/tests/test_runtime.py

```
@@ -0,0 +1,541 @@
+            patch('rocketride._engine.manager.spawn_engine', new_callable=AsyncMock, return_value=12345),
+            patch('rocketride._engine.manager.wait_healthy', new_callable=AsyncMock),
+            patch.object(mgr, '_register_signal_handlers'),
+        ):
+            uri, we_started = await mgr.ensure_running()
+
+        assert we_started is True
+        assert uri == 'http://127.0.0.1:5570'
+        mock_db.register.assert_called_once()
+
+    @pytest.mark.asyncio
+    async def test_teardown_when_we_started(self, tmp_path):
```

Ruff (0.15.7)

[warning] Unused method argument: `tmp_path`

(ARG002)